community/modules/file-system/nfs-server/scripts/install-nfs-server.sh.tpl

```diff
@@ -12,14 +12,18 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set -ex
 
-yum -y install nfs-utils
 systemctl start nfs-server rpcbind
```

Comment:
shouldn't these not start after the exports are configured and the folder mounted?

Reply:
It seems to work just fine, also there is `exportfs -r` as a last line in the script.
Do you think it's worth to move it down?